bin/cml.js

```diff
@@ -89,13 +107,30 @@ const handleError = (message, error) => {
           description: 'Maximum log level',
           choices: ['error', 'warn', 'info', 'debug'],
           default: 'info'
+        },
+        driver: {
+          type: 'string',
+          choices: ['github', 'gitlab', 'bitbucket'],
+          defaultDescription: 'infer from the environment',
+          description: 'Git provider where the repository is hosted'
+        },
+        repo: {
+          type: 'string',
+          defaultDescription: 'infer from the environment',
+          description: 'Repository URL or slug'
+        },
+        token: {
+          type: 'string',
+          defaultDescription: 'infer from the environment',
+          description: 'Personal access token'
         }
       })
       .fail(handleError)
       .middleware(setupOpts)
       .middleware(setupLogger)
       .middleware(setupTelemetry)
-      .commandDir('./cml', { exclude: /\.test\.js$/ })
```

Comment:
Do not needed?

Reply:
Absolutely needed; otherwise, it won't pick subcommands (in the `cml` directory)

Reply:
If needed why is removed? 🤔

Reply:
Oh, you mean just the `exclude` part, not the whole `commandDir` call. 😅 Now the `cml` directory doesn't contain any test files, so there is nothing to exclude.